activerecord/test/cases/adapters/sqlite3/quoting_test.rb

```diff
@@ -58,22 +58,6 @@ def test_type_cast_false
           assert_equal 0, @conn.type_cast(false, c)
         end
 
-        def test_type_cast_string
-          assert_equal '10', @conn.type_cast('10', nil)
-
-          c = Column.new(nil, 1, Type::Integer.new)
-          assert_equal 10, @conn.type_cast('10', c)
-
-          c = Column.new(nil, 1, Type::Float.new)
-          assert_equal 10.1, @conn.type_cast('10.1', c)
-
-          c = Column.new(nil, 1, Type::Binary.new)
-          assert_equal '10.1', @conn.type_cast('10.1', c)
-
-          c = Column.new(nil, 1, Type::Date.new)
-          assert_equal '10.1', @conn.type_cast('10.1', c)
```

Comment:
Oh, now I understand it. So those tests still make sense because the column information is still being used differently from 5.0, but this assertion needs to be changed or removed